source/kraken/fill_disruption_from_chaos.cpp

```diff
@@ -108,10 +108,46 @@ make_severity(const chaos::Severity& chaos_severity, nt::new_disruption::Disrupt
     return std::move(severity);
 }
 
+static boost::optional<nt::new_disruption::LineSection>
+make_line_section(const chaos::PtObject& chaos_section,
+                  nt::PT_Data& pt_data,
+                  const boost::shared_ptr<nt::new_disruption::Impact>& impact) {
+    if (!chaos_section.has_pt_line_section()) {
+        LOG4CPLUS_WARN(log4cplus::Logger::getInstance("log"),
+                       "fill_disruption_from_chaos: LineSection invalid!");
+        return boost::none;
+    }
+    const auto& pb_section = chaos_section.pt_line_section();
+    nt::new_disruption::LineSection line_section;
+    auto* line = find_or_default(pb_section.line().uri(), pt_data.lines_map);
+    if (line) {
+        line_section.line = line;
+    } else {
+        LOG4CPLUS_WARN(log4cplus::Logger::getInstance("log"),
+                       "fill_disruption_from_chaos: line id in LineSection invalid!");
```

Comment:
maybe it would be easier to debug with the line id in the log (same for the stops id)